app/models/customer.py

```diff
@@ -0,0 +1,24 @@
+from flask import current_app
+from app import db
+
+
+class Customer(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime, nullable=True)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+    rentals = db.relationship('Rental', back_populates='customer')
+
+    def to_json(self):
+        return  {
+        "id": self.id,
+        "name": self.name,
+        "postal_code": self.postal_code,
+        "phone": self.phone,
+        "registered_at": self.registered_at,
+        "videos_checked_out_count": self.videos_checked_out_count
+        }
```

Comment:
You can use `rentals` to compute the `videos_checked_out_count` though you need to add a field to `Rental` to keep track of if it's been checked in:

```suggestion
    rentals = db.relationship('Rental', back_populates='customer')

    def to_json(self):
        return  {
        "id": self.id,
        "name": self.name,
        "postal_code": self.postal_code,
        "phone": self.phone,
        "registered_at": self.registered_at,
        "videos_checked_out_count": len([rental for rental in self.rentals if not rental.check_in_date])
        }
```